src/Redux/Slice/authSlice.js

```diff
@@ -0,0 +1,43 @@
+import { createSlice } from '@reduxjs/toolkit';
+import { login } from 'Redux/Thunk';
+import { signup } from 'Redux/Thunk';
+
```

Comment:
You can write this in a single line `import { login, signup } from 'Redux/Thunk';`

Reply:
Thanks.. I will refactor it